app/src/main/java/org/mifos/mobilebanking/ui/fragments/AccountsFragment.java

```diff
@@ -367,11 +372,14 @@ public void filterShareAccount(List<CheckboxStatus> statusModelList) {
      */
     @Override
     public void showError(String errorMessage) {
-        ll_error.setVisibility(View.VISIBLE);
-        rvAccounts.setVisibility(View.GONE);
-        noAccountText.setText(errorMessage);
-        noAccountIcon.setImageResource(R.drawable.ic_error_black_24dp);
-        Toast.makeText(getActivity(), errorMessage, Toast.LENGTH_SHORT).show();
+        if (!Network.isConnected(getContext())) {
+            rvAccounts.setVisibility(View.GONE);
```

Comment:
@miPlodder I don't think this line make any sense.
`
 rvAccounts.setVisibility(View.GONE);
` 
because we are passing `rvAccount` in showSweetNoInternetUI().. method to visibility GONE.